### ISSUE TITLE
Not compatible with Enderal animations

### ISSUE THREAD

**Issue by larcturus** [ShikyoKira/Project-New-Reign---Nemesis-Main#229]
It appears that since sureAI used FNIS to create their custom animations during "cutscenes" that nemesis isn't picking them up for whatever reason so these animations won't play unless I'm using FNIS.

**Comment by ShikyoKira** [comment]
did you correctly point to the correct directory that's in nemesis.ini?

**Comment by larcturus** [comment]
I did.
This is what's in my .ini file

SkyrimDataDirectory=D:\Games\Steam\steamapps\common\Enderal\Data\
MaxAnimation=20000
first=false
width=528
height=750
modNameWidth=380
authorWidth=75
priorityWidth=47

**Comment by ShikyoKira** [comment]
oh, try run nemesis with enderal FNIS patch as well

**Comment by larcturus** [comment]
That's the first thing I tried, lol. No dice

I can see that Nemesis recognizes the patch when I run the engine, but the cutscenes still don't play out. I have a theory that it has to do with the PCEA aspect of it not applying the new animations to the NPCs and that it only applies the animations to my character. I tried just running the Nemesis engine from my directory after removing all the PCEA stuff that I got from the Nexus page but Nemesis won't run without the PCEA stuff.

**Comment by ShikyoKira** [comment]
Nemesis won't run without PCEA? You could need a reinstall of Nemesis. Something must be missing there

**Comment by M0J0MAN** [comment]
How did you get Nemesis to run with the Endereal install directory?
Mine crashes on update engine as it doesn't recognize the directory.

**Comment by larcturus** [comment]
I'm not sure if you're using a mod manager or not, but I never had any problems with running nemesis. I was using the steam store version of enderal and manually downloaded to my data folder by following all of the instructions on the nemesis PCEA page on the Nexus site. Keep in mind that I never did sort out the issue of my NPC cutscene animations not playing in enderal. I eventually just switched back to using FNIS because of this.

**Comment by M0J0MAN** [comment]
I'm on MO2 using the steam version.
I got an error the first time I ran nemesis as it doesn't recognize the custom Enderal directory.
Every time I attempt to update engine it crashes soon after it detects the directory, another thread on github has someone else bring up the same problem.
Also if you're still trying to get it to work the FNIS Enderal patch has some hiccups unless you install Enderal's animations unextracted as a mod, maybe that will help Nemesis as well though I have no idea.

**Comment by larcturus** [comment]
Hmm. Yea I'm not sure. I remember there being a lot of issues with running MO2 and Nemesis/FNIS so because I was only going to do one play-through and not switch mods up that much I just used Vortex. 

ANYWAYS. like I said before you're best bet is to run Nemesis directly from the DATA folder by installing it manually. If you're still having issues then I know some issues are cause by certain mods getting confused between the skyrim folder and the Enderal Folder. I had a similar issue when trying to run the creation kit for Enderal. This video below was made for people trying to run creation kit with enderal but it will essentially show you how to sort of link both the skyrim and enderal directory folders so anything trying to access the assets of one will be able to access the assets of the other. It might fix your issue.

https://www.youtube.com/watch?v=dQJVP7OH9MU

Also I finished the game without too much issue using FNIS and have since uninstalled everything so I'm not actively working with any of this stuff anymore.
